app/controllers/providers_controller.rb

```diff
@@ -1,9 +1,8 @@
 class ProvidersController < ApplicationController
-  before_action :build_recruitment_cycle, only: :index
   rescue_from JsonApiClient::Errors::NotFound, with: :not_found
 
   def index
-    @providers = @recruitment_cycle.providers
+    @providers = Provider.where(year: Settings.current_cycle).all
```

Comment:
Won't this mean that new providers in the next cycle might not be able to get to their organisation?